2.4/root/usr/libexec/httpd-prepare

```diff
@@ -20,6 +20,10 @@ if [ -v HTTPD_SCL ] ; then
   ln -s /var/www /opt/rh/httpd24/root/var/www
 fi
 
+if head "/etc/redhat-release" | grep -q "^Red Hat Enterprise Linux release 8"; then
```

Comment:
I've seen more cases where we need to behave differently on different OS versions, so maybe we could have some env. variables defined in s2i-core, that we can use. It is not something I'd block this PR for, it is reported as a suggestion in https://github.com/sclorg/s2i-base-container/issues/182.